pkg/webhook/webhook.go

```diff
@@ -199,6 +199,9 @@ func getNamespaceFromOwnerReference(ownerRef metav1.OwnerReference) (namespace s
 				break
 			}
 		}
+	default:
+		glog.Infof("owner reference kind is not supported: %v", ownerRef.Kind)
+		return
```

Comment:
shouldn't namespace set to default before returning it ?

Reply:
I was thinking there is not much different return empty or default namespace, but I think at least it can cover the case where pod is created in default namespace. I will make the change.

Reply:
Done